create.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>CREATE</h1>
+    </header>
+
+    <main>
+      <section class="text--field">
+        <lable for="text--field__question">Creat a tricky question?:</lable>
```

Comment:
I guess you mean `<label>`
```suggestion
        <label for="text--field__question">Create a tricky question?:</label>
```